repository-ydb-v2/src/main/java/tech/ydb/yoj/repository/ydb/client/YdbValidator.java

```diff
@@ -103,6 +103,42 @@ public static void validate(String request, StatusCode statusCode, String respon
         }
     }
 
+    public static boolean isServerSideError(StatusCode statusCode) {
+        return switch (statusCode) {
```

Comment:
switch is better that check contains in Set because this switch will produce compile error if StatusCode have new status